web/components/LandingPage.js

```diff
@@ -1,124 +1,92 @@
+                <TeamList />
 
-                    <Flex direction='row'>
-                        <Card 
-                            bg='bottomColor' 
-                            height='75px'
-                            width='100%'
-                            mt='27rem'
-                            mr='-15.6rem'
-                        >
+                <NavBarContainer>
+                    <NavigationBar></NavigationBar>
+                </NavBarContainer>
 
-                        </Card>
-
-                        <Card
-                            bg='grey'
-                            height='73px'
-                            width='250px'
-                            mt='27.1rem'
-                            ml='-0.1rem'
-                            borderRadius={5}
-                        >
-                            <Flex direction='row' justifyContent='flex-start'>
-                                <Card
-                                    onClick={this.goToPage}
-                                    bg='grey'
-                                    height='73px'
-                                    width='125px'
-                                    borderRadius={5}
-                                >
-                                    <Card
-                                        bg='black'
-                                        width='1px'
-                                        height='61px'
-                                        my='5%'
-                                        mx='100%'
-                                        depth='-2px 0px 10px rgba(0,0,0,0.5)'
-                                    >
-                                    </Card>
-                                </Card>
-                                <Card
-                                    onClick={this.goToPage}
-                                    bg='grey'
-                                    height='73px'
-                                    width='125px'
-                                    borderRadius={5}
-                                >
-                                    <Card
-                                        bg='black'
-                                        width='1px'
-                                        height='61px'
-                                        my='5%'
-                                        mx='1%'
-                                        depth='2px 0px 10px rgba(0,0,0,0.5)'
-                                    >
-                                    </Card>
-                                </Card>
-                            </Flex>
-                        </Card>
-                    </Flex>
-                </Box>
-            </div>
+            </LandingPageContainer>
         );
     }
 }
+
+const LandingPageContainer = styled(Box)`
+    position: absolute;
+    top: 0px;
+    left: 0px;
+    min-width: 375px;
+    width: 100%;
+    min-height: 100%;
+    height: 100%;
+    display: flex;
```

Comment:
Use the `Flex` component instead of using these styles directly to take advantage of reusability. (we will go through the Flex component in the workshop today @FedeMadoery)